File: Ecole--Edtech1/src/ForumBundle/Entity/Sujet.php
<?php

namespace ForumBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
/**
 * Sujet
 *
 * @ORM\Table(name="sujet")
 * @ORM\Entity(repositoryClass="ForumBundle\Repository\SujetRepository")
 */
class Sujet
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var int
     *
     * @ORM\Column(name="createur", type="integer")
     */
    private $createur;

    /**
     * @var string
     *
     * @ORM\Column(name="titre", type="string", length=255)
     *
     * @Assert\Length(min=10, minMessage="Titre doit contenir au moins 10 caracteres")
     * @Assert\Length(max=50, maxMessage="Titre doit contenir au maximum 50 caracteres")
     */
    private $titre;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255)
     */
    private $description;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime")
     */
    private $date;

    /**
     * @var int
     *
     * @ORM\Column(name="score", type="integer")
     */
    private $score;

    /**
     * @var int
     *
     * @ORM\Column(name="vues", type="integer")
     */
    private $vues;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set createur
     *
     * @param integer $createur
     *
     * @return Sujet
     */
    public function setCreateur($createur)
    {
        $this->createur = $createur;

        return $this;
    }

    /**
     * Get createur
     *
     * @return int
     */
    public function getCreateur()
    {
        return $this->createur;
    }

    /**
     * Set titre
     *
     * @param string $titre
     *
     * @return Sujet
     */
    public function setTitre($titre)
    {
        $this->titre = $titre;

        return $this;
    }

    /**
     * Get titre
     *
     * @return string
     */
    public function getTitre()
    {
        return $this->titre;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Sujet
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return Sujet
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set score
     *
     * @param integer $score
     *
     * @return Sujet
     */
    public function setScore($score)
    {
        $this->score = $score;

        return $this;
    }

    /**
     * Get score
     *
     * @return int
     */
    public function getScore()
    {
        return $this->score;
    }

    /**
     * @return int
     */
    public function getVues()
    {
        return $this->vues;
    }

    /**
     * @param int $vues
     */
    public function setVues($vues)
    {
        $this->vues = $vues;
    }


}

File: ProjetPi/src/com/mycompany/myapp/entities/Sujet.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author admin
 */
public class Sujet {

    private int sujet_id;
    private int createur_id;
    private String titre;
    private String description;
    private String date;
    private int score;
    private int vues;

    public Sujet() {

    }

    public Sujet(int sujet_id, int createur_id, String titre, String description, String date, int score, int vues) {
        this.sujet_id = sujet_id;
        this.createur_id = createur_id;
        this.titre = titre;
        this.description = description;
        this.date = date;
        this.score = score;
        this.vues = vues;
    }

    public Sujet(int createur_id, String titre, String description, String date, int score, int vues) {
        this.createur_id = createur_id;
        this.titre = titre;
        this.description = description;
        this.date = date;
        this.score = score;
        this.vues = vues;
    }

    public int getSujet_id() {
        return sujet_id;
    }

    public int getCreateur_id() {
        return createur_id;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

    public int getVues() {
        return vues;
    }

    public void setVues(int vues) {
        this.vues = vues;
    }

    public void setSujet_id(int sujet_id) {
        this.sujet_id = sujet_id;
    }

    public void setCreateur_id(int createur_id) {
        this.createur_id = createur_id;
    }

    @Override
    public String toString() {
        return "Sujet{" + "sujet_id=" + sujet_id + ", createur_id=" + createur_id + ", titre=" + titre + ", description=" + description + ", date=" + date + ", score=" + score + ", vues=" + vues + '}';
    }

}
